components/TextFields/src/MDCTextInputControllerLegacyDefault.m

```diff
@@ -602,18 +624,18 @@ - (BOOL)needsUpdatePlaceholderAnimationConstraintsToUp {
 
   UIOffset offset = [self floatingPlaceholderOffset];
 
-  CGFloat leadingConstant = [self floatingPlaceholderAnimationConstraintLeadingConstant:insets
-                                                                                 offset:offset];
-  CGFloat trailingConstant = [self floatingPlaceholderAnimationConstraintTrailingConstant:insets
-                                                                                   offset:offset];
+  CGFloat leadingConstant =
+      [self floatingPlaceholderAnimationConstraintLeadingConstant:insets offset:offset];
+  CGFloat trailingConstant =
+      [self floatingPlaceholderAnimationConstraintTrailingConstant:insets offset:offset];
 
   return self.placeholderAnimationConstraintLeading.constant != leadingConstant &&
-  self.placeholderAnimationConstraintTrailing.constant != trailingConstant;
+         self.placeholderAnimationConstraintTrailing.constant != trailingConstant;
 }
 
 // When placeholder is not up, it just uses the layout code that comes for free from the text field
 // itself. That means these constraints need go away. So, we can use the existence of them as a
-// way to check if the placeholder is floating up. See isPlacehodlerUp.
+// way to check if the placeholder is floating up. See isplaceholderUp.
```

Comment:
Nit: camelCase `isPlaceholderUp` to match the method

Reply:
Don't know how that one happened.